### DISCUSSION TITLE
Navbar

### DISCUSSION THREAD

**Discussion by Sahad-09** [saadeghi/daisyui#1896] [Q&A] [answered]
Are navbars not responsive by default ???

**Comment by saadeghi** [comment, marked as answer]
It's not a good idea for the navbar itself to be responsive because there are infinite ways to make a navbar depending on the content (logo, menu, buttons, dropdown, text, icons, search field...) and where to position each one (left, center, right) and how we want them to be placed on mobile. Even sometimes we need to completely restructure things on mobile (for example moving a search field from navbar to sidebar or transforming menu items from "text + icon" to "icon only"

So if I provide a few specific ways to design a navbar. it would limit the design choices completely.

Instead, we provide positioning options. You can position items at start, end, or center. Then you can decide what to show/hide on different screens using Tailwind CSS utility classes. (something like `hidden lg:block` to show it only on desktop. Or `lg:hidden` to show it only on mobile)

This is the most efficient and customizable way
